doc/whatsnew/3/3.0/index.rst

```diff
@@ -6,30 +6,26 @@
    :maxdepth: 2
 
 :Release: 3.0.0
-:Date: TBA
+:Date: 2023-10-02
 
 
 Summary -- Release highlights
 =============================
 
-Note Bene for alpha users: Do not upgrade unless you want to
-test things out. Some changes will require a configuration change,
-that won't be compatible with 2.x. For example some extensions
-were removed in favor of a builtin message (``compare-to-zero``
-and ``compare-to-empty-string``). We'll make a configuration
-upgrade tool available before 3.0.0 is out.
+Pylint now support python 3.12 officially.
 
-Pylint now provides some important usability and performance improvements,
-along with enacting necessary breaking changes and long-announced deprecations.
-
-There's going to be frequent beta releases,
-before the official releases, everyone is welcome to try the betas
-so we find problems before the actual release.
+This long anticipated major version also provides some important usability
+and performance improvements, along with enacting necessary breaking changes
+and long-announced deprecations. The documentation of each message with an
+example is very close too.
```

Comment:
pyreverse also got a lot of love but hard to pinpoint a particular issues that was super important compared to the others.